inference.py

```diff
@@ -0,0 +1,32 @@
+from loader import Preprocessor, StandardConvSuite
+import torch
+
+
+class INFPreprocessor(Preprocessor):
+    def __init__(self, device: torch.device):
+        """
+        Wrapper for `Iterator` class that preprocesses compressed data to pytorch tensors.
+        :param device: device for tensors
+        """
+        super(Preprocessor, self).__init__()
+        self.device = device
+
+
+class INF(StandardConvSuite):

```

Comment:
INF looks like it's an abbreviation and may make people unneccessairly wonder what it stands for, why not just Inference or PositionEmbeddingInference?